pulp_smash/pulp3/pytest_plugin/__init__.py

```diff
@@ -14,14 +16,21 @@
 from pulp_smash import cli
 from pulp_smash.api import _get_sleep_time
 from pulp_smash.config import get_config
+from pulp_smash import cli as smash_cli
 from pulp_smash.pulp3.bindings import delete_orphans, monitor_task
 from pulp_smash.pulp3.fixture_utils import add_recording_route
 
-from pulpcore.client.pulpcore import ApiClient, TasksApi
+from pulpcore.client.pulpcore import ApiClient, StatusApi, TasksApi
+from pulpcore.client.pulpcore.exceptions import ApiException
 
 
 cfg = get_config()
+cli = smash_cli.Client(cfg)
```

Comment:
Maybe blind, but i don't see this used. Also it collides with the import on line 16.

Reply:
copy-paste from the previous place of residence, I'll remove it